suite-native/module-home/src/components/EmptyPortfolioCrossroads.tsx

```diff
@@ -56,17 +68,21 @@ export const EmptyPortfolioCrossroads = () => {
         <VStack spacing="medium" flex={1}>
             <Card style={applyStyle(cardStyle, { flex: 2 })}>
                 <VStack spacing="large" justifyContent="center" alignItems="center">
-                    <Pictogram
-                        icon="trezor"
-                        variant="green"
-                        title={translate(
-                            'moduleHome.emptyState.connectOrImportCrossroads.gotMyTrezor.title',
-                        )}
-                        subtitle={translate(
-                            'moduleHome.emptyState.connectOrImportCrossroads.gotMyTrezor.description',
-                        )}
-                        size="large"
-                    />
+                    <VStack alignItems="center" spacing="large">
```

Comment:
This should me stretched across the screen as its in develop

Reply:
I am not sure what you mean here.

Reply:
![image](https://github.com/trezor/trezor-suite/assets/36101761/1ecbdf21-230b-4a2a-bb8e-f920181c735d)

On develop right now, it's stretched across the screen and now it seems to have shrinked due to styling changes